src/views/home/browseOurTop/BrowserTop.tsx

```diff
@@ -0,0 +1,54 @@
+import { FaEye } from "react-icons/fa";
+import { Link } from "react-router-dom";
+import { TopRatings } from "./topRatings";
+
+export const BrowseOurTop = () => {
+  return (
+    <div className="container mt-40">
+      <div className="flex justify-center">
+        <h2 className="text-4xl font-bold">Browse Our Top</h2>
+        <span className="text-dark-blue italic text-4xl font-bold ml-2">
+          Niche
+        </span>
+      </div>
+      <div>
+        <p className="text-lg mt-11 max-w-[600px] text-center m-auto">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit aspernatur
+          illum vel sunt libero voluptatum repudiandae veniam maxime tenetur
+          fugiat eaque alias nobis doloremque culpa nam.
+        </p>
+      </div>
+      <div className="flex flex-row justify-center mt-11">
+        <nav className="flex">
+          <ul className="flex items-center justify-center p-2">
+            <li className="bg-[#f5f5f5] p-4  lg:w-80 md:w-64 sm:w-60 text-center text-[#0044bb] border-b-2 border-b-blue-500">
```

Comment:
Please check out tailwind config, we have specified these colors as variables, so you can use for example: bg-gray-chalk